File: src/main/java/com/gmail/borlandlp/minigamesdtools/arena/gui/providers/GUICreatorHub.java
package com.gmail.borlandlp.minigamesdtools.arena.gui.providers;

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider;
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub;

public class GUICreatorHub extends CreatorHub {
    public GUIProvider createGuiProvider(String ID, AbstractDataProvider dataProvider) throws Exception {
        return (GUIProvider) this.create(ID, dataProvider);
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/arena/gui/providers/GUICreatorHub.kt
package com.gmail.borlandlp.minigamesdtools.arena.gui.providers

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub

class GUICreatorHub : CreatorHub() {
    @Throws(Exception::class)
    override fun create(itemID: String, dataProvider: AbstractDataProvider): GUIProvider {
        return super.create(itemID, dataProvider) as GUIProvider
    }
}